lib/glimesh_web/live/user_settings/components/channel_settings_live.ex

```diff
@@ -18,4 +20,20 @@ defmodule GlimeshWeb.UserSettings.Components.ChannelSettingsLive do
     |> assign(:channel_delete_disabled, session["channel_delete_disabled"])}
   end
 
+  @impl true
+  def handle_event("rotate_stream_key", _params, socket) do
+    with :ok <- Bodyguard.permit(Glimesh.Streams, :update_channel, socket.assigns.channel.user, socket.assigns.channel) do
+      case Streams.rotate_stream_key(socket.assigns.channel.user, socket.assigns.channel) do
+        {:ok, changeset} ->
+          {:noreply,
+          socket
+          |> put_flash(:info, "Stream key reset")
+          |> assign(:channel_changeset, Streams.Channel.changeset(changeset))}
+        {:error, changeset} ->
```

Comment:
sad unused changeset
```suggestion
        {:error, _changeset} ->
```